posthog-core/test/posthog.flush.spec.ts

```diff
@@ -19,19 +19,15 @@ describe('PostHog Core', () => {
 
     it("doesn't fail when queue is empty", async () => {
       jest.useRealTimers()
-      await expect(posthog.flush()).resolves.toEqual([])
+      await expect(posthog.flush()).resolves.not.toThrow()
     })
 
     it('flush messages once called', async () => {
```

Comment:
I guess this is why we were returning, it was useful for testing purposes. Can we assert we flushed all 3 elements here? Maybe inside the `mocks.fetch` expect?

Reply:
See `successfulMessages` below